### PR TITLE
[docs]fix the bug of missing tag in infiniteScroll.md,upload.md and change the options api to composition api in avatar docs.

### PR DESCRIPTION
**Pull request by Ther-su**
fix the bug of missing tag

### CODE REVIEW

**Comment by zouhangwithsweet** [comment]
In fact, I don’t think it is necessary to change the documentation of all components to VCA.We should have certain trade-offs.
E.g `notification`, we use the new api to implement. The doc should update to VCA. The common components' docs may not be necessary to change.

File: website/docs/zh-CN/avatar.md [hunk 1]
```diff
@@ -108,14 +113,15 @@
...
 <script>
-  export default {
-    data() {
+  import { defineComponent, ref } from 'vue'
+  export default defineComponent({
+    setup() {
       return {
         fits: ['fill', 'contain', 'cover', 'none', 'scale-down'],
-        url: 'https://fuss10.elemecdn.com/e/5d/4a731a90594a4af544c0c25941171jpeg.jpeg'
+        url: ref('https://fuss10.elemecdn.com/e/5d/4a731a90594a4af544c0c25941171jpeg.jpeg')
```
Comment (JeremyWuuuuu) [inline website/docs/zh-CN/avatar.md:121]: No need to set this reactive.

File: website/docs/zh-CN/avatar.md [hunk 2]
```diff
@@ -31,15 +31,16 @@
...
-  export default {
-    data () {
+  import { defineComponent, ref, reactive } from 'vue'
+  export default defineComponent({
+    setup () {
       return {
-        circleUrl: "https://cube.elemecdn.com/3/7c/3ea6beec64369c2642b92c6726f1epng.png",
-        squareUrl: "https://cube.elemecdn.com/9/c2/f0ee8a3c7c9638a54940382568c9dpng.png",
+        circleUrl: ref("https://cube.elemecdn.com/3/7c/3ea6beec64369c2642b92c6726f1epng.png"),
+        squareUrl: ref("https://cube.elemecdn.com/9/c2/f0ee8a3c7c9638a54940382568c9dpng.png"),
```
Comment (JeremyWuuuuu) [inline website/docs/zh-CN/avatar.md:39]: Same, no need to mark this as reactive, since it does not change.

**Comment by Ther-su** [comment]
@zouhangwithsweet  In fact, I don't know what is VCA.It is difficult to find the information about VCA in Search Engines. I think I can understand what you say better if you can tell me the detail about VCA. Thank you.

**Comment by Ther-su** [comment]
@JeremyWuuuuu Finished.

**Comment by zazzaz** [comment]
Thanks

**Review by zazzaz — APPROVED** [review]

**Pull request merged**